### PR TITLE
Black dark theme

### PR DESCRIPTION
**Pull request by IvLaptev**
Adds additional dark theme. Thit theme is a little bit darker than the dark theme.

image
image

### CODE REVIEW

**Comment by IvLaptev** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by IvLaptev** [comment]
recheck

File: static/themes/night.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  --error-button-background-color: #3867d6;
+  --error-button-color: #ffffff;
+  --submit-button-border-color: #1a46ad;
+  --submit-button-background-color: #3867d6;
+  --submit-button-color: #ffffff;
+  --graph-opacity: 1;
+  /* --graph-filter: brightness(0.35) sepia(1) hue-rotate(88deg) saturate(8); */
+}
+
+@media (max-width: 500px) {
```
Comment (AnandChowdhary) [inline static/themes/night.css:28]: You can remove this since it's now merged to `master` and will be auto-added to each theme.

**Review by AnandChowdhary — CHANGES_REQUESTED** [review]

**Comment by IvLaptev** [comment]
There was one more usless line.

**Review by AnandChowdhary — APPROVED** [review]

**Pull request merged**